.github/workflows/deploy-data-plane-controller.yaml

```diff
@@ -1,39 +1,35 @@
 name: Deploy data-plane-controller
 
 on:
-  push:
-    # Only on pushes to the master branch
+  workflow_run:
+    workflows: ["Platform Build"]
+    types:
+      - completed
     branches: [master]
-    # Only run if function code or the ci config itself has changed
-    paths:
-      - crates/data-plane-controller/**
-      - .github/workflows/deploy-data-plane-controller.yaml
-
-env:
-  CARGO_INCREMENTAL: 0 # Faster from-scratch builds.
+  workflow_dispatch: {}
 
 jobs:
-  build:
+  deploy:
     runs-on: ubuntu-24.04
+    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
     permissions:
-      # Permissions required of the Github token in order for
-      # federated identity and authorization to work.
       contents: read
       id-token: write
+      packages: read
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
-          submodules: true
-          lfs: true
-
-      - uses: supabase/setup-cli@v1
-      - run: supabase start
-
-      - name: Build `data-plane-controller`
-        run: cargo build --release -p data-plane-controller
 
-      - run: mv target/release/data-plane-controller crates/data-plane-controller/
+      - name: Get image tag
+        id: image-tag
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            TAG="dev-next"
+          else
+            TAG=$(git describe --tags --always)
```

Comment:
this may require `depth: 0` on the actions/checkout to pull tags? Otherwise i thought it was shallow

Reply:
Good catch!